rootfs/scheduler/__init__.py

```diff
@@ -142,7 +142,7 @@
                 "value":"$id"
             },
             {
-                "name":"DEIS_RELEASE",
+                "name":"WORKFLOW_RELEASE",
```

Comment:
it's used so that the application can gain knowledge of its own release information. It's important!